projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -914,6 +935,10 @@ const setFormError = ( form, invalidFields, opts = {} ) => {
 	}
 
 	const count = invalidFields.length;
+	if ( ! count && !! L10N[ opts.type ] ) {
```

Comment:
Lets leave a comment here for future self why we are doing this.

Reply:
Sure.